### ISSUE TITLE
[bug] Can't re-enable 'show public notice' switch

### ISSUE THREAD

**Issue by ylfzq** [shadowsocks/shadowsocks-manager#405]
Please answer these questions before submitting your issue. Thanks!

### What version of shadowsocks-manager are you using?
0.15.4

### What operating system are you using?
ubuntu 16.04 x64

### What version of Node.js are you using?
v6.14.1

### What did you do?
Admin role, in Settings-Group Management-Choose one group-show public notice switch, I can't re-enable this switch after I disabled it before.

### What did you expect to see?
Can re-enable this switch

### What did you see instead?
Tell me success, but with this switch nothing changed 

### What is your config in detail (with all sensitive info masked)?
I think it has nothing to do with the config file

Thank you for reading this issue and thank you for developing this wonderful software.

**Comment by gyteng** [comment]
I think 0.15.3 has already fix the problem

2018-05-02 15:30 GMT+08:00 lux.feary <notifications@github.com>:

**Comment by ylfzq** [comment]
You're right, I can't recur this issue in a new opened browser in incognito mode(Google Chrome). 

I guess may be something wrong in my cookie. What was amazing, I can't got my traffic statistics on Server page also. But both of them came to be ok after my cleaning of cookie. Or does it have something to do with 'Service Worker‘, may be it's a cache problem?

You can close this issue if you're quite sure it's not a problem, thank you.
